src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -135,6 +135,9 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
                                 if (row.original.partyCode === vulnerableVotes.partyCode) {
                                     return <div className="has-background-dark has-text-white">{row.value}</div>;
                                 }
+                                if (row.value === 1) {
+                                    return "null";
+                                }
```

Comment:
This is incorrect. You need to check for the winning party, not for the value.

Reply:
```
     if (row.original.partyCode === vulnerableVotes.winner.partyCode) {
              return "null";
         }
```

Reply:
I think this should work. 👍